tests/utils.go

```diff
@@ -1221,6 +1226,48 @@ func cleanupSubresourceServiceAccount() {
 	}
 }
 
+func createPodPreset(ppName, volName, podName, srcPath, dstPath string) {
```

Comment:
Please use the full name of the variables - at least for ppName => podPresetName .

I also find it more readable to be explicit about the types, but it could be just me; feel free to ignore me on this.

Finally, I think your 'podName' is really not a pod name, but a pod *label* instead, isn't it ? e.g. `virt-launcher-<vmi name>` is the name of a virt-launcher pod. At least, I that's what I understood from the [docs](https://kubernetes.io/docs/concepts/workloads/pods/podpreset/#how-it-works).

Reply:
answered in a sep comment, since github had problems, changed parts according my understanding